org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/resources/DefaultSonarLintProjectsProvider.java

```diff
@@ -52,17 +52,17 @@ public Collection<ISonarLintProject> get() {
   /** This should be extended with further checks from other plug-ins providing technical projects */
   private static boolean isTechnicalProject(String projectName) {
     // check for Eclipse PDE technical projects
-    var isTechnicalProject = TECHNICAL_PROJECT_PDE.equals(projectName) && Platform.getBundle("org.eclipse.pde.ui") != null;
+    var isTechnicalProject = projectName.contains(TECHNICAL_PROJECT_PDE) && Platform.getBundle("org.eclipse.pde.ui") != null;
 
     // check for Eclipse PDT technical projects
     isTechnicalProject = isTechnicalProject ||
       (TECHNICAL_PROJECT_PDT.equals(projectName) && Platform.getBundle("org.eclipse.php.ui") != null);
```

Comment:
What about PDT?

Reply:
The technical project is only called `Servers` and if we just check if a project name contains this, this will rule out more projects by accident I think. The other ones are called so cryptically, that I doubt anyone will call their project like this, also because it might not even work.